### DISCUSSION TITLE
Regex Replace with line breaks

### DISCUSSION THREAD

**Discussion by gazoir** [mProjectsCode/obsidian-meta-bind-plugin#424] [Q&A]
Hi all! I'm struggling to work out the implementation of regexp in metabind. I'm trying to do the following:

1. Specify a section with a start and end using any unique searchable identifier -- for example `###START###` and `###END###`. There will be multiple line breaks and an unspecified amount of content in between those identifiers. Could be 2 lines, or as much as 20 I suppose.
2. Replace the entire section with new content, including line breaks and all characters in between the identifiers.

So far I can successfully find and replace single lines using the ###START### identifier, but I'm trying to tell Metabind to replace **everything** in between the start and end tags - I've tried using `*` and `.` as wildcards but I think they're ignoring line breaks. I gather there's a flag I can provide in most regexp implementations but I'm struggling to pass it to Metabind. If anyone has done this and can help I'd be dead grateful. Thanks!

**Comment by mProjectsCode** [comment]
You can specify custom regexp flags, see [the docs](https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/reference/buttonactions/regexpreplaceinnote/).

Other that that I recommend using something like https://regexr.com/ to write and test regexps.

**Reply by gazoir** [reply]
Thanks I'll check it out
